src/insns/cjalr_jalr_32bit.adoc

```diff
@@ -24,11 +24,20 @@ Capability Mode Description::
 CJALR allows unconditional, indirect jumps to a target capability. The
 target capability is obtained by unsealing `cs1` and incrementing its address by the
 sign-extended 12-bit immediate, and then setting the least-significant bit of the
-result to zero. The target capability may have
+result to zero. The target address may then have
 xref:section_invalid_addr_conv[xrefstyle=short]
-performed and is then installed in <<pcc>>. The <<pcc>>
-of the next instruction following the jump (<<pcc>> + 4) is sealed and written
-to `cd`.
+performed. The check for length violation is done next using the converted
+address and the original bounds.
```

Comment:
I think this text is better placed as a note in the `Exceptions::` section where the actual failure cases are highlighted.